front/pages/poke/[wId]/index.tsx

```diff
@@ -62,27 +65,30 @@ export const getServerSideProps = withSuperUserAuthRequirements<{
   }
 
   // TODO(2024-02-28 flav) Stop fetching agent configurations on the server side.
-  const [dataSources, agentConfigurations, subscriptionModels] =
-    await Promise.all([
-      getDataSources(auth, { includeEditedBy: true }),
-      (async () => {
-        return (
-          await getAgentConfigurations({
-            auth,
-            agentsGetView: "admin_internal",
-            variant: "full",
-          })
-        ).filter(
-          (a) =>
-            !Object.values(GLOBAL_AGENTS_SID).includes(
-              a.sId as GLOBAL_AGENTS_SID
-            )
-        );
-      })(),
-      Subscription.findAll({
-        where: { workspaceId: owner.id },
-      }),
-    ]);
+  const [
+    dataSources,
+    dataSourceViews,
+    agentConfigurations,
+    subscriptionModels,
+  ] = await Promise.all([
+    getDataSources(auth, { includeEditedBy: true }),
+    DataSourceViewResource.listByWorkspace(auth),
```

Comment:
Can we avoid doing this? This page is already pretty slow to load today, I added a table for data source views yesterday https://github.com/dust-tt/dust/pull/7551 that uses SWR hooks instead.

Reply:
More plumbing. It's poke. Won't fix :p